jena-arq/src/test/java/org/apache/jena/sparql/core/AbstractDatasetGraphTests.java

```diff
@@ -328,6 +328,26 @@
         assertFalse(dsg.containsGraph(gn)) ;
     }
 
+    @Test public void clear_02() {
+        DatasetGraph dsg = emptyDataset();
+        dsg.add(new Quad(NodeFactory.createURI("g1"),
```

Comment:
FYI: no need to change anything:

For writing test data, you can use "SSE.parse...."

```
dsg.add(SSE.parseQuad("(<g1> <a1> <b1> <c1>)"));
```

The default graph is "_":

```
dsg.add(SSE.parseQuad("(_ <a0> <b0> <c0>)"));
```